readme.txt

```diff
@@ -71,6 +71,13 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 
 == Changelog ==
 
+= 1.7.2 11/19/2020 =
+
+- Fix: Load wc-tracks in the homepage notice admin script. #5638
+- Fix: Link component prop caused a React warning. #5653
+- Fix: Flickering of order panel while loading. #5655
+- Fix: Tax code duplicated when clicking button to remove. #5661
+
 = 1.7.0 11/11/2020 =
```

Comment:
Can we add an ` ( unreleased ) ` after the date here, just to signal it was never uploaded to WP.org?

Reply:
Done here: https://github.com/woocommerce/woocommerce-admin/pull/5685/commits/6260cfa6546e868d578fd3631155c0fa666bf13c

Thanks :)